_components/banner.md

```diff
@@ -26,16 +26,16 @@ lead: The banner identifies your site as an official website of the United State
       </ul>
       <h4>When to consider something else</h4>
       <ul class="usa-content-list">
-        <li><strong>If you use neither https nor a .gov/.mil domain.</strong> The banner text calls out https and .gov/.mil domains as ways to communicate that this is an official government site. Don't use the banner if your site doesn't do those things.</li>
-        <li><strong>Any time it would be misleading.</strong> The banner should be used to reduce confusion. Avoid using the banner on sites meant only for testing or experiments, or are otherwise not meant to be seen as an official website.</li>
+        <li><strong>If you use neither HTTPS nor a .gov/.mil domain.</strong> The banner text indentifies HTTPS and .gov/.mil domains as indicators that a website is an official government website. Use the banner only if your site uses both the proper TLD <em>and</em> HTTPS.</li>
+        <li><strong>Any time it would be misleading.</strong> The banner should be used to reduce confusion. Avoid using the banner on any site meant only for testing or otherwise not meant to be identified as an official government website.</li>
       </ul>
       <h4>Usability guidance</h4>
       <ul class="usa-content-list">
-        <li><strong>Show the banner on every site page.</strong> Use the gov banner at the top of every page of a site. It can be confusioing or misleading if it appears on some site pages and no others.</li>
-        <li><strong>Don't use more than one.</strong> Don't use multiple banners at the top of a page to communicate multiple identities, like an agency banner, and a gov banner. Consolidate into a single banner instead.</li>
-        <li><strong>Keep the text up-to-date.</strong> Use the most current version of the banner. Use the "Here's how you know" text from the most current version and avoid modification.</li>
-        <li><strong>Use the proper TLD.</strong> If your project uses a .mil top-level domain, change the text in the "Here's how you know" section to "The .mil means it's official."</li>
+        <li><strong>Use the provided text without customization.</strong> The banner is most effective as an identifier and a learning tool when its message is consistent across government websites. With only a few exceptions (see Implementation guidance, below), sites should use the TLD-appropriate text provided, unaltered. Use the Spanish version of the banner for Spanish-language websites.</li>
+        <li><strong>Use the version appropriate to your website’s TLD.</strong> If your project uses a .mil top-level domain, use the .mil banner text.</li>
+        <li><strong>Show the banner on every site page.</strong> Use the gov banner at the top of every page of a site. It can be confusing or misleading if it appears on some site pages and not others.</li>
```

Comment:
Remove `gov banner` and replace with `banner`. Use `page` instead of `site page`.

<li><strong>Show the banner on every page.</strong> Use the banner at the top of every page of a site. It can be confusing or misleading if it appears on some pages and not others.</li>